<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>426</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="showOfflineContacts">
       <property name="text">
        <string>Show offline contacts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showGroups">
       <property name="text">
        <string>Show groups</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="contactsList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
